apps/__init__.py

```diff
@@ -40,10 +40,33 @@ def register_extensions(app):
 
 def register_blueprints(app):
     for module_name in ('authentication', 'home', 'api', 'k8s', 'cli'):
-        module = import_module('apps.{}.routes'.format(module_name))
+        module = import_module(f'apps.{module_name}.routes')
         app.register_blueprint(module.blueprint)
-    # Load socketio endpoints
-    import_module("apps.events")
+
+    # (PoC) ContainerLabs: registra blueprint se habilitado
+    enable_clabs = app.config.get(
+        "ENABLE_CLABS",
+        str(os.getenv("ENABLE_CLABS", "false")).lower() in ("1", "true", "yes", "on")
+    )
+    if enable_clabs:
+        try:
+            clabs = import_module("apps.clabs.routes")
+            app.register_blueprint(clabs.blueprint)
+            app.logger.info("ContainerLabs blueprint registrado (ENABLE_CLABS=true).")
+        except Exception as e:
+            app.logger.error(f"Falha ao registrar ContainerLabs: {e}")
+    else:
+        app.logger.info("ContainerLabs desativado (ENABLE_CLABS=false).")
+
+    # Carrega endpoints do Socket.IO (xterm)
+    # -> Pula no Windows porque usa pty/termios/fcntl (só Unix)
+    try:
+        if os.name != "nt":
+            import_module("apps.events")
+        else:
+            app.logger.warning("Skipping apps.events on Windows (no termios/pty).")
+    except Exception as e:
+        app.logger.warning(f"Skipping apps.events due to error: {e}")
```

Comment:
well, the `apps.events` is crucial for some of the most important features of the platform, so this seems like a workaround just for allowing the development environment to be operational on Windows. If that is the case, I suggest you include on the development environment for windows a requirement for Docker. Then, the developer would leverage a docker Unix container to actually have the app operational.